app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :comments, foreign_key: 'user_id'
   has_many :likes, foreign_key: 'user_id'
   has_many :posts, foreign_key: 'user_id'
 
-  validates :name, presence: true
-  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+  # validates :name, presence: true
+  # validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly remove these unnecessary commented code from your codebase so as to achieve clean codes.